pkg/api/v1alpha1/provider_spec.go

```diff
@@ -284,3 +288,8 @@ type GCPServiceAccount struct {
 	// account.
 	Scopes []string `json:"scopes"`
 }
+
+type GPUConfig struct {
+	AcceleratorType string `json:"acceleratorType"`
+	CountPerNode    int64  `json:"countPerNode"`
```

Comment:
`CountPerNode` is not required, you can consider renaming it to:  `Count` or `AcceleratorCount`. It will be derived based on the context under which it is defined. If defined under `Worker` it will apply to a single worker, if defined under `WorkerPool` or `Workers` then it will apply to the entire set of workers in the worker pool.